server/controllers/auth.js

```diff
@@ -9,11 +10,19 @@ authRouter.get("/google", passport.authenticate("google", { scope: ["profile", "
 // @route GET /auth/google/callback
 authRouter.get("/google/callback", passport.authenticate("google", { failureRedirect: "/" }), 
     (request, response) => {
-        response.json({
-            "status":"success",
-            "message": "user authenticated!"
-        }).end();
+        // response.json({
+        //     "status":"success",
+        //     "message": "user authenticated!"
+        // }).end();
+        // response.send(request.user);
```

Comment:
here, i believe this works right? `request.user` exists and is what you expect?

if so, notice that this router has 3 parameters. the 2nd param is the auth middleware where is appends `user` to `request`